misc/thesis-fields.tex

```diff
@@ -15,10 +15,11 @@
 \supervisionEnglish{This research was carried out under the supervision of Prof.~Big Shot, in the Faculty of Computer Science.}
 \supervisionHebrew{המחקר בוצע בהנחייתו של פרופסור אישחשוב עצמוני, בפקולטה למדעי המחשב.}
 
-\GregorianDateEnglish{January 2012}
-\GregorianDateHebrew{ינואר \textenglish{2012}}
-\JewishDateEnglish{Tevet 5772}
-\JewishDateHebrew{טבת התשע"ב}
+% Date to be used in the title page. Even though the month is printed, we need the
+% day to calculate the Hebrew date.
+\year=2012
```

Comment:
These need to be package options. Indicating when the thesis was submitted can't be the result of some arbitrary override, it's an important detail.

Reply:
Do you think if we change `\today`, but use package options for that, it would be better?

E.g.,

```latex
    \year=\iitthesis@GregorianDateYear
    ...
```